frontend/packages/console-app/src/components/nodes/NodeDetailsOverview.tsx

```diff
@@ -31,29 +32,30 @@ const NodeDetailsOverview: React.FC<NodeDetailsOverviewProps> = ({ node }) => {
     name: node.metadata.name,
     namespace: node.metadata.namespace,
   });
+
   return (
     <div className="co-m-pane__body">
-      <SectionHeading text="Node Details" />
+      <SectionHeading text={t('nodes~Node details')} />
```

Comment:
@rhamilto Quick question: The SectionHeadings are in sentence case. Do we want to apply  sentence case to all (title or heading) strings in console?